File: XDownloader/app/src/main/java/com/pv/xdownloader/ui/commonview/BottomBar.java
package com.pv.xdownloader.ui.commonview;

import android.annotation.SuppressLint;
import android.content.Context;
import android.util.AttributeSet;
import android.view.LayoutInflater;
import android.view.View;

import com.pv.xdownloader.R;

public class BottomBar extends View {

    public BottomBar(Context context, AttributeSet attrs) {
        super(context, attrs);
        initView(context);
    }

    @SuppressLint("InflateParams")
    private void initView(Context context) {
        LayoutInflater layoutInflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        View rootView = layoutInflater.inflate(R.layout.layout_bottom_bar, null);

    }


}

File: XDownloader/app/src/main/java/com/pv/xdownloader/ui/commonview/BottomBar.kt
package com.pv.xdownloader.ui.commonview

import android.annotation.SuppressLint
import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import android.view.View

import com.pv.xdownloader.R

class BottomBar(context: Context, attrs: AttributeSet) : View(context, attrs) {

    init {
        initView(context)
    }

    @SuppressLint("InflateParams")
    private fun initView(context: Context) {
        val layoutInflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
        val rootView = layoutInflater.inflate(R.layout.layout_bottom_bar, null)

    }


}
